### ISSUE TITLE
Editor Output Scrollbar not appearing while running the project

### ISSUE THREAD

**Issue by nEsTiCaL** [godotengine/godot#78836]
### Godot version

v4.1.rc1.official [1f9e540f1]

### System information

Windows 10

### Issue description

Hello,

while running the Project from Editor and printing something with print() the Output Tab simply overflows and stays put. The Scrollbar does not appear and it is also not scrolling to the newes output.
Even after closing the current running project there is no scrollbar and i cant force the scrolling with e.g. marking the text.

Only after switching Tabs between Output and Debugger it is appearing and the output seems complete.

### Steps to reproduce

have a project with a print. run it in editor. try scrolling the output Tab while project is running, try scrolling after closing it (still no Scrollbar). Switch Tabs and see the Scrollbar apeearing :)

### Minimal reproduction project

N/A

**Comment by AThousandShips** [comment]
Duplicate of: #78434

**Comment by akien-mga** [comment]
I now pinned #78434 so it's easier to find, especially since we're about to merge a fix so it will be closed, but still reproducible in RC 1 until we release RC 2.

**Comment by nEsTiCaL** [comment]
i havent connected the RichtTextLabel with my Error - sorry!
